employeeapplication/src/main/java/com/artimes/employee/controller/EmployeeController.java

```diff
@@ -0,0 +1,22 @@
+package com.artimes.employee.controller;
+
+import com.artimes.employee.model.Employee;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class EmployeeController {
+
+    @PostMapping("/employeeDetails")
+//    @ResponseBody
+    public ResponseEntity<String> employeeDetails(@RequestBody Employee employee){
```

Comment:
Please change the name of this method to createEmployee as you are creating an employee.
Please update the request mapping path too!

Reply:
Changed method name and request mapping to createEmployee .
Please verify and let me know if any changes required.